FAQ.md

```diff
@@ -38,37 +38,34 @@ languages.
 
 # Licensing
 
-Slint is available under two licenses:
+Slint can be used under either a [commercial license](./LICENSES/LicenseRef-Slint-commercial.md) or [GNU GPLv3](./LICENSES/GPL-3.0-only.txt), at your choice.
 
- * GPLv3, for the growing ecosystem of Free and Open Source Software.
- * Commercial, for use in closed-source projects. See <https://slint-ui.com/#offering>
-
-The commercial license is free if you help us promote Slint: Check out our
+The commercial license can be provided for free if you help us promote Slint: Check out our
 [ambassador license](https://slint-ui.com/ambassador-program.html).
 
-## What is the commercial licensing model?
+## What are the commercial licensing options?
 
-The basic principle behind our commercial licensing is that you start for free and pay when you're shipping.
+We offer 3 options under the commercial license - Ambassador, Flex and Buyout. All 3 options include a non-exclusive, perpetual, irrevocable, non-transferable right to use Slint.
 
-We offer a perpetual license option and we generally deploy a per-product license, regardless of how many developers, designers, Q&A engineers are using Slint.
+The Ambassador license is a free license that can be provided in lieu of helping us promote Slint.
 
-If this doesn't fit you, don't hesitate to contact us and we'd be happy to work together to find a solution.
+The Flex license is a per User-Seat license (with updates included as long as the subscription is active) to develop apps with Slint.
+Distribution of such apps requires additional fees.
 
-You can find a more detailed overview of our commercial licensing and the pricing at <https://slint-ui.com/pricing.html>.
+The Buyout license is a volume-based buyout license that includes unlimited User-Seats (with updates included) to develop apps with Slint as well as distribute such apps up to the purchased volume.
 
 ## Can I try Slint using the GPL version and then switch to the commercial license later?
 
-Yes. The GPL is a distribution license that applies only when you ship your application. You can
-evaluate Slint and develop your product internally using the GPL license, and only acquire a commercial
-license when you want to ship your product. If you choose a per seat licensing model, the time spent
-developing needs to be accounted for. However, support for bug fixes requires a commercial license.
+Yes. You can evaluate Slint using the GPL license, and acquire the commercial license after the evaluation, with the option to transfer the development work from GPL to commercial for free.
```

Comment:
For consistency, but perhaps we should use "acquire" above instead of "obtain" hmm.

```suggestion
Yes. You can evaluate Slint using the GPL license, and obtain the commercial license after the evaluation, with the option to transfer the development work from GPL to commercial for free.
```

Reply:
Yeah. I mean, I'm a little torn. I think "acquire" puts the emphasis on the act of getting the license, while "obtain" emphasises the state after getting it a little more.

Differently angle: "get" is the easiest, "obtain" is a little harder, and "acquire" requires the most effort of "getting" the license.